### DISCUSSION TITLE
Commits

### DISCUSSION THREAD

**Discussion by Clirussa** [INGCOM-UNRN-P1/cursada-2023#94] [Preguntas y respuestas] [answered]
Hola, estaba hablando con una compa y me comentó que tenemos que hacer commits. Miro ese apartado y tengo todo esto que no se de donde salió y no se que tengo que hacer, como limpiarlo o si es solo de este TP1 que estaba haciendo. 

![commit](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/102642611/70852b2b-47e1-4aea-b3f9-dfa63291dbd0)

**Comment by martinvilu** [comment]
Si lo limpias, vas a romper algo de tu computadora.

En donde hiciste el clone?, esto se hace en una carpeta aislada, y esa es la que tenés que abrir en el vscode.

Porque veo demasiados archivos, y algunos son se sistema

**Reply by Clirussa** [reply]
Lo que tengo es de Python, que es la que no está aislada. Lo de C lo tengo en el escritorio. ¿Entonces que hago?

**Reply by martinvilu** [reply, marked as answer]
Cambia de directorio de trabajo al repositorio.

Porque es una **muy** mala idea crear un repositorio en la raíz  de tu usuario.

Hace clone en un lugar aislado y cambia a ese lugar
